src/coupling/src/adapter/4C_coupling_adapter_converter.cpp

```diff
@@ -138,26 +138,25 @@ bool Coupling::Adapter::MatrixLogicalSplitAndTransform::operator()(
         exporter_ = std::make_shared<Core::LinAlg::Export>(permsrcmap, src.row_map());
       }
 
-      std::shared_ptr<Epetra_CrsMatrix> permsrc =
-          std::make_shared<Epetra_CrsMatrix>(::Copy, permsrcmap.get_epetra_map(), 0);
-      int err = permsrc->Import(*src.epetra_matrix(), exporter_->get_epetra_export(), Insert);
+      auto permsrc = std::make_shared<Core::LinAlg::SparseMatrix>(permsrcmap.get_epetra_map(), 0);
```

Comment:
I think you could also construct a `SparseMatrix` from a `Core::LinAlg::Map`:
```suggestion
      auto permsrc = std::make_shared<Core::LinAlg::SparseMatrix>(permsrcmap, 0);
```
See https://4c-multiphysics.github.io/4C/doxygen/classCore_1_1LinAlg_1_1SparseMatrix.html#a89da60a44a536fe1729c4e9f6e7bd5c5.